pylintrc

```diff
@@ -0,0 +1,445 @@
+[MASTER]
+
+# A comma-separated list of package or module names from where C extensions may
+# be loaded. Extensions are loading into the active Python interpreter and may
+# run arbitrary code
+extension-pkg-whitelist=
+
+# Add files or directories to the blacklist. They should be base names, not
+# paths.
+ignore=examples
```

Comment:
This pull request ignores the `examples` directory for now, since they're probably going to be refactored up later on. It doesn't make sense to invest the effort to clean up code that we're just going to radically change.